app/src/main/java/com/github/geekarist/henripotier/PotierApplication.java

```diff
@@ -0,0 +1,43 @@
+package com.github.geekarist.henripotier;
+
+import android.app.Application;
+import android.util.Log;
+
+import com.squareup.okhttp.OkHttpClient;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import retrofit.Callback;
+import retrofit.RestAdapter;
+import retrofit.RetrofitError;
+import retrofit.client.OkClient;
+import retrofit.client.Response;
+import retrofit.http.GET;
+
+public class PotierApplication extends Application {
+
+    private HenriPotier mRestAdapter;
+
+    @Override
+    public void onCreate() {
```

Comment:
Piste : tu mets une variable static PotierApplication et dans ton constructeur tu fais instance = this;
Après tu rajoutes une méthode static pour récupérer ton instance : PotierApplication.instance(); 
Ca t'évitera le cast tout moche dans ton activité. Un truc important c'est que ta classe PotierApplication, le système Android te garantit que ca sera instancié qu'une fois!